applications/GeoMechanicsApplication/custom_constitutive/thermal_filter_law.h

```diff
@@ -0,0 +1,61 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Mohamed Nabi
+//                   John van Esch
+//
+
+#pragma once
+
+#include "includes/constitutive_law.h"
+
+namespace Kratos
+{
+
+class RetentionLaw;
```

Comment:
I do not see a reference on RetentionLaw here. Is it possible to remove this line?